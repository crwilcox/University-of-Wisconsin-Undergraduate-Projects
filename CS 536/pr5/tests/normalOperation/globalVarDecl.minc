int i = 1;
double d = 3.1415;

void main()
{
	i++;
	d = d + 1.0;
	return;
}
