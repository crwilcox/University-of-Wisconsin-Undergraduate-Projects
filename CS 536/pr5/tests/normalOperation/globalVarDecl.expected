int i = 1;
double d = 3.1415;
void main() {
  (i++); /* int */
  (d = (d + 1.0)); /* double */
  return;
}

